modules/javafx.graphics/src/main/java/com/sun/glass/ui/headless/HeadlessView.java

```diff
@@ -0,0 +1,130 @@
+package com.sun.glass.ui.headless;
+
+import com.sun.glass.events.ViewEvent;
+import com.sun.glass.ui.Pixels;
+import com.sun.glass.ui.View;
+import java.util.Map;
+
+public class HeadlessView extends View {
+
+    private Map capabilities;
+    private int x = 0;
+    private int y = 0;
+    private long parentPtr = 0;
+    private Pixels pixels;
+
+    private boolean imeEnabled;
+
+    @Override
+    protected void _enableInputMethodEvents(long ptr, boolean enable) {
+        this.imeEnabled = enable;
+    }
+
+    @Override
+    protected long _create(Map capabilities) {
+        this.capabilities = capabilities;
+        return 1;
+    }
+
+    @Override
+    protected long _getNativeView(long ptr) {
+        return ptr;
+    }
+
+    @Override
+    protected int _getX(long ptr) {
+        return x;
+    }
+
+    @Override
+    protected int _getY(long ptr) {
+        return y;
+    }
+
+    @Override
+    protected void _setParent(long ptr, long parentPtr) {
+        parentPtr = parentPtr;
```

Comment:
You're assigning a variable to itself. But even if you qualify `this.parentPtr`, the assignment seems pointless.

Reply:
Added this -- I agree it looks pointless now, but preparing for follow-up review comments where the parentPtr is prepared.